### PR TITLE
Feature/mvp

### PR DESCRIPTION
**Pull request by lbeckman314**
Minimal viable product branch. Features Click CLI and batch async URL signing and downloading.

### CODE REVIEW

File: drs_downloader/main.py [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+    Args:
+        tsv_path (str): The input file containing a list of DRS URI's.
+        drs_header (str): Column header for the DRS URI's.
+
+    Returns:
+        List[str]: The URI's corresponding to the DRS objects.
+    """
+
+    urls = []
+    df = pd.read_csv(Path(tsv_path), sep='\t')
```
Comment (bwalsh) [inline drs_downloader/main.py:38]: See https://docs.python.org/3/library/csv.html
```
with open(Path(tsv_path)) as file:
    tsv_file = csv.reader(file, delimiter="\t")
```

**Comment by lbeckman314** [comment]
Ah I will update the test cases to reflect the new changes and then send an update!

**Comment by bwalsh** [comment]
```
---------- coverage: platform darwin, python 3.9.12-final-0 ----------
Name                         Stmts   Miss  Cover
------------------------------------------------
drs_downloader/__init__.py       0      0   100%
drs_downloader/download.py      83     52    37%
drs_downloader/logger.py         3      0   100%
drs_downloader/main.py         162    107    34%
------------------------------------------------
TOTAL                          248    159    36%
```

`37%` is pretty low.  ok for now.  let's improve it next version.

**Comment by bwalsh** [comment]
Remove `.DS_Store` files

**Pull request closed without merging**